openc3/lib/openc3.rb

```diff
@@ -22,6 +22,9 @@
 
 # This file sets up using the OpenC3 framework
 
+# Improve DNS (especially on Alpine)
+require 'resolv-replace'
```

Comment:
Doesn't this need to be added to the Gemfile?

Reply:
At least in ruby 3.1 its part of the stdlib